css/sticky.scss

```diff
@@ -0,0 +1,109 @@
+
+.sticky__heading{
+    text-align: center;
+    color: var(--secondary-700);
+    font-size: 2.4rem;
+    font-weight: 700;
+}
+
+#sticky-notes-container{
+    display: flex;
+    padding: 2rem ;
+    border: 2px solid gray;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    min-height: 100vh;
+    gap: 2rem;
+ 
+}
+
+.sticky__card{
+    padding: 2rem;
+    display: flex;
```

Comment:
Did you look into nesting in scss? Probably that's something you can use here?

Reply:
Yes, used in latest ones.